packages/snap-shared/src/MockData/MockData.ts

```diff
@@ -42,7 +42,7 @@ export class MockData {
 
 	constructor(config?: MockDataConfig) {
 		this.initialConfig = config || defaultConfig;
-		this.updateConfig(this.initialConfig);
+		if (config) this.updateConfig(config);
```

Comment:
This seems to change the functionality a bit - it no longer uses the defaultConfig.